back/app/finders/ideas_finder.rb

```diff
@@ -23,8 +23,28 @@ class IdeasFinder < ApplicationFinder
     Idea.unscoped.where(id: ids).order_as_specified(id: ids)
   }
 
+  def initialize(params, scope: nil, includes: [], current_user: nil, paginate: true)
+    scope ||= _base_scope
+    super(
+      params,
+      scope: only_idea_inputs_scope(scope),
+      includes: includes,
+      current_user: current_user,
+      paginate: paginate
+    )
+  end
+
   private
 
+  def only_idea_inputs_scope(scope)
+    phase_ideas_with_creation_phase = scope.where(creation_phase: Phase.where(participation_method: %w[ideation budgeting]))
```

Comment:
Fine for me to remove it, but that means that if the creation phase of an input is set to an ideation/budgeting phase, the input will not be included.

Reply:
Perhaps we might want to add a model-level validation to ensure that the creation_phase cannot have the participation_method "ideation" or "budgeting", to avoid this issue to silently slip through in the future?